setup.cfg

```
@@ -1,6 +1,6 @@
 [metadata]
 name = bring-api
-version = 0.8.1
+version = 0.8.1a1

```

This is also possible since setuptools 46.4.0

```suggestion
version = attr: package.__version__

```

---

Then we need to bump `pyproject.toml`, no?
```
[build-system]
requires = [
    "setuptools>=42",
    "wheel"
]
```

---

bumped it to `>=46.4.0`